app/controllers/guests_controller.rb

```diff
@@ -0,0 +1,18 @@
+class GuestsController < ApplicationController
+  def show
+    @guest = Guest.find(params[:id])
+  end
+
+  def add_room
+    @guest = Guest.find(params[:id])
+    room = Room.find(params[:room_id])
+    @guest.rooms << room
```

Comment:
Yup, this works, great job with the logic. As your apps get larger, you'll want to move even small logic like this to the models, could make a method for it called Guest#add_room(room) that way the Model is technically the one that is adding this record to the linking table in the db.